common/commands/src/validator/mixnet/query/query_all_gateways.rs

```diff
@@ -15,7 +15,7 @@ pub struct Args {
 }
 
 pub async fn query(args: Args, client: &QueryClientWithValidatorAPI) {
```

Comment:
client is called `QueryClientWithValidatorAPI` -> I guess it should also be renamed